src/components/Locator.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div id='locator'></div>
+</template>
+
+<script>
+export default {
+  name: 'userLocate',
+  // is this how to register button to appear in the #locator?
+  template: '<button onclick="getLocation()">i am here</button>',
+  data () {
+    return {
+      // userLocate: {}
+    }
+  },
+  methods: {
+    // why is compiler saying 'let is a reserved word?' when I'm trying to make it a variable?
+    let locatorDiv = document.getElementById('locator')
```

Comment:
This needs to either be inside of a method or be it's own method that returns the locatorDiv.

In keeping with simple, testable functions, I'd probably go with the latter here.